routes/main.py

```diff
@@ -424,3 +424,11 @@ def suggest(limit=10):
         suggestions = [{"value": (prefix + p[0] + "?"), "data": ""} for p in q]
 
     return better_jsonify(suggestions=suggestions)
+
+
+@routes.route("/translate")
+@cache.cached(timeout=30 * 60, key_prefix="translate", query_string=True)
```

Comment:
Why is this page cached?

Reply:
Since translations are unlikely to change for a given query string? "When True, the cache key  used will be the result of hashing the ordered query string parameters." @haukurb ?

Reply:
Yes, but this is the UI page, not the API page.